_includes/breadcrumbs.html

```diff
@@ -0,0 +1,45 @@
+<nav aria-label="Breadcrumb" class="breadcrumb">
+    <div class="container">
+        <!-- <ol>
+            <li class="breadcrumb-item"><a href="/">Home</a></li>
+            <li class="breadcrumb-item active" aria-current="page">Mentorship</li>
+        </ol> -->
+        <!-- <div clas="breadcrumb-back__wrapper">
+            <div class="breadcrumb__back">
+                <div class="material-symbols-outlined" aria-hidden="true">arrow_back</div>
+                <a href="/blog">Back to blog</a>
+            </div>
+        </div> -->
+
+        
```

Comment:
You can use this to know the current path and later use if to check if it is this page and show the breadcrumbs for that and this will solve your problem.  

```
 {% assign current_path = page.url | remove: '.html' | remove_first: '/' }
```